frontend/src/pages/dashboard/betas/create/index.tsx

```diff
@@ -24,13 +26,25 @@ export default function BetaCreate() {
     );
     try {
       const cloudflareVideoUid = await uploadBetaVideoToCloudfare(beta.video as File);
-      await postCreateBeta({
+      const { data } = await postCreateBeta({
```

Comment:
Prefer useSafeRequest for several reasons: 
- auto logout if unauthorized 
- for consistency (refer to how it is implemented in CreateEditJioForm)
- onSuccess and onError actions implementation is cleaner

Reply:
I think this change is non-trivial for chained promises. Whether it's cleaner is also up for debate, for post requests case, since we are not using the state.